### PR TITLE
feat: limit logs to 7gb when run with docker compose

### PR DESCRIPTION
**Pull request by deodad**
Docker Compose will store an indefinite amount of logs. Long running hubs will eventually run out of disk space.

This will limit logs to 7 1gb size files when run via docker compose.

---

## PR-Codex overview
This PR focuses on adding logging options to the `docker-compose.yml` file for the `hubble` app.

### Detailed summary
- Removed the `./.hub` and `./.rocks` volume mappings.
- Added logging options for the `hubble` app:
  - Set the logging driver to "json-file".
  - Added options for maximum log file size (`max-size`) and maximum number of log files (`max-file`).

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/hubble/docker-compose.yml [hunk 1]
```diff
@@ -11,3 +11,8 @@ services:
     volumes:
       - ./.hub:/home/node/app/apps/hubble/.hub
       - ./.rocks:/home/node/app/apps/hubble/.rocks
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "1g"
+        max-file: "7"
```
Comment (sanjayprabhu) [inline apps/hubble/docker-compose.yml:18]: Even 2 gigs should be fine.

**Review by sanjayprabhu — APPROVED** [review]
Thanks

File: apps/hubble/docker-compose.yml (hunk 1 above)
Comment (varunsrin) [reply apps/hubble/docker-compose.yml:18]: how much log history is 7 GB? i think if we have history for ~ 7 days during a normal sync (excluding initial sync) that should be sufficient

Comment (varunsrin) [reply apps/hubble/docker-compose.yml]: @deodad i updated to 2x1 gb and merged it so we could get it into this release.

**Pull request merged**